DigitalLearningSolutions.Web/Controllers/RegisterController.cs

```diff
@@ -138,29 +141,58 @@ public IActionResult Summary()
         [HttpPost]
         public IActionResult Summary(SummaryViewModel model)
         {
+            var data = TempData.Peek<DelegateRegistrationData>()!;
             if (!ModelState.IsValid)
             {
-                var data = TempData.Peek<DelegateRegistrationData>();
-                var viewModel = MapToSummary(data!);
+                var viewModel = MapToSummary(data);
                 viewModel.Terms = model.Terms;
                 return View(viewModel);
             }
 
-            // TODO HEEDLS-396 create new user and redirect to confirmation page
-            return View(model);
+            var baseUrl = ConfigHelper.GetAppConfig()["AppRootPath"];

```

Comment:
I think we should use the "CurrentSystemBaseUrl" here to make sure the email links to the old system. We will change it once we rebuild the approval page :)